web/cypress/integration/consumer/create-consumer-with-limit-count-plugin-form.spec.js

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create and delete consumer with limit-count plugin form', () => {
+  beforeEach(() => {
+    cy.login();
+
+    cy.fixture('selector.json').as('domSelector');
+    cy.fixture('data.json').as('data');
+  });
+
+  const selector = {
+    count: '#count',
+    time_window: '#time_window',
+    redis_timeout: '#time_window',
+    key: '#key',
+    rejected_code: '#rejected_code',
+    policy: '#policy',
+    redis_host: '#redis_host',
+    redis_port: '#redis_port',
+    redis_password: '#redis_password',
+    redis_database: '#redis_database',
+    redis_timeout: '#redis_timeout',
+    redis_cluster_name: '#redis_cluster_name',
+    redis_cluster_nodes_0: '#redis_cluster_nodes_0',
+    redis_cluster_nodes_1: '#redis_cluster_nodes_1',
```

Comment:
why we need these two fields?

Reply:
Because limit-count plugin schema rule:  must be two redis_cluster_nodes